packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4455,7 +4455,12 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     if (_dependencies != null && _dependencies!.isNotEmpty) {
       final List<DiagnosticsNode> diagnosticsDependencies = _dependencies!
         .map((InheritedElement element) => element.widget.toDiagnosticsNode(style: DiagnosticsTreeStyle.sparse))
```

Comment:
Sort while you still have a List<Widget> and you can use toStringShort or just the runtime type of the Widget class which would also be reasonable.

Reply:
good idea. done.